resources/log4j.properties

```diff
@@ -20,5 +20,6 @@ log4j.appender.metabase=metabase.logger.Appender
 log4j.logger.metabase=INFO
 log4j.logger.metabase.db=DEBUG
 log4j.logger.metabase.driver=DEBUG
+log4j.logger.metabase.middleware=DEBUG
```

Comment:
reminder to remove this before merging.  I'd recommend that you exclude these from your commits as this is really just something for you personally while you are developing.